src/main/java/org/symphonyoss/symphony/messageml/elements/MessageML.java

```diff
@@ -155,4 +161,19 @@ public void setChime(boolean chime) {
   public String getPresentationMLTag() {
     return PRESENTATIONML_TAG;
   }
+
+  /**
+   * If the messageML contains a uiAction with a target-id this method checks that exists a corresponding
+   * dialog element with the same id.
+   */
+  private void validateTargetIdForUIActions() throws InvalidInputException {
+    List<Element> uiActionWithTargetId = getChildren().stream()
```

Comment:
can it be nested several level down? (like a ui action inside a container? like a form or a div?)

Reply:
you are right, we should search also for nested UIActions.